docs/src/pages/ui/primitives/select/index.page.mdx

```diff
@@ -145,17 +139,51 @@ import '@aws-amplify/ui-react/styles.css';
     <Select
       defaultValue=""
       placeholder="Please select a fruit"
-      variation="filled"
+      variation="flushed"
     >
       <option value="apple">Apple</option>
       <option value="banana">Banana</option>
       <option value="orange">Orange</option>
     </Select>
-    <Select
-      defaultValue=""
-      placeholder="Please select a fruit"
-      variation="flushed"
-    >
+  </Flex>
+</Example>
+
+### Placeholder
+
+You can set a placeholder option out of box via the `placeholder` props. The value for the placeholder option will set to empty string `""`. If you would like to pass your own placeholder with a custom value, then just simply leave it off and pass your custom one.
+
+```jsx
+import { Select } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+
+import { Select } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+
+// use the placeholder out of box
+<Select defaultValue="" placeholder="Please select a fruit">
+  <option value="apple">Apple</option>
+  <option value="banana">Banana</option>
+  <option value="orange">Orange</option>
+</Select>;
+
+// or pass a custom placeholder
+<Select defaultValue="">
```

Comment:
Would anything be selected here since the `defaultValue`is not DEFAULT?

Reply:
I forgot to update the value here but the first option will always be selected if no matching value found